### ISSUE TITLE
After training, building the 3DBall game out to stand alone folder does not work

### ISSUE THREAD

**Issue by JohnnyPhoton** [Unity-Technologies/ml-agents#193]
I trained the 3D ball model, and set the brain to internal. All runs fine in the editor, but when I build the game the way I normally would it does not work. The tiles don't move and the balls just fall down. The folder contains the usual exe, data folder, and player dll. I checked the data folder to make sure the libtensorflow dll was there. What am I missing? Thanks for this tool, it's been challenging but fun!

**Comment by awjuliani** [comment]
Hi @JohnnyPhoton,

I haven't come across this issue before. It is strange that it runs fine using the internal brain in the editor, but not when built. Can you give some more information on the platform you are using?

**Comment by JohnnyPhoton** [comment]
@awjuliani 

Thanks for getting back to me. Ha, great it's just me! I wanted to put it on my Surface Pro to show to people over the holiday. Now however, I'm just trying to run it on the same machine I trained it on. It's running Windows 10 pro build 16299 with an i7-6700, 32 Gigs of ram,  MSI Z170A gaming M7 motherboard, and a GTX 1070. I'm using Unity 2017.3.0f3.

O.K. I just installed Unity 2017.3.0f3 on the Surface Pro 4 and copied the project over to it. Same thing, runs in the editor just fine but when I build and run it, the balls just fall to the floor. I'd like to figure this out but at least now I can show it to people running in the editor.

Must be something about my projects settings? I think the only thing I changed was enabling the resolution/quality menu at startup.
